assets/js/script.js

```diff
@@ -22,3 +22,77 @@ document.querySelectorAll('#menu_link').forEach((btn) => {
 });
 
 // Menu
+
+// Speakers
+
+const speakers = [
+  {
+    id: '1',
+    name: 'Yochai Benkler',
+    image: './assets/images/person1.jpg',
+    position: 'Professor at Harvard computer science school',
+    description: 'Have the time to share your thought and options with expects for each topic',
+  },
+  {
+    id: '2',
+    name: 'James',
+    image: './assets/images/person2.jpg',
+    position: 'Professor at University of Cambridge',
+    description: 'Have the time to share your thought and options with expects for each topic',
+  },
+  {
+    id: '3',
+    name: 'Thomas',
+    image: './assets/images/person3.jpg',
+    position: 'Professor at Yale computer science school',
+    description: 'Have the time to share your thought and options with expects for each topic',
+  },
+  {
+    id: '4',
+    name: 'David',
+    image: './assets/images/person4.jpg',
+    position: 'Professor at Standford University',
+    description: 'Have the time to share your thought and options with expects for each topic',
+  },
+
+];
+
+speakers.forEach((speaker) => {
```

Comment:
- [ ] I kindly suggest that you wrap this function is a window.onload function so that it will be executed only  when the web page has completely loaded all content